packages/pancake-uikit/src/components/SubMenuItems/SubMenuItems.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Box } from "../Box";
+import MenuItem from "../MenuItem/MenuItem";
+import StyledSubMenuItems from "./styles";
+import { SubMenuItemsProps } from "./types";
+
+const MenuItems: React.FC<SubMenuItemsProps> = ({ items = [], ...props }) => {
+  return (
+    <StyledSubMenuItems justifyContent={["start", "start", "center"]} {...props} pl={["12px", "12px", "0px"]}>
```

Comment:
When the style remains the same across breakpoints it can be set as null, so `["start", null, "center"]` - although obviously what you have here is fine, and maybe you prefer it like this?

Reply:
Oh wasn't aware of that, will use this syntax then.